modules/ingester/instance.go

```diff
@@ -81,9 +87,22 @@ type instance struct {
 	hash hash.Hash32
 }
 
+type searchStreamingBlockEntry struct {
+	b   *search.StreamingSearchBlock
```

Comment:
do StreamingSearchBlock and SearchBlock follow the same pattern as AppendBlock and BackendBlock?

the first is meant to just append data to as it comes in and the second is for storing in some kind of completed format ready for upload? maybe i just need to go read the block code :)